src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: 'AKIARUBOPCTSSU3W7SML',
+    secretAccessKey: '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',
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist) {
+    const messageBody = [
+      {
+        DateTime: Date.now().toString(),
+        Purpose: {
+          type: purpose,
+          subtype: purposeParams,
+          params: {
+            platform: redirectTo,
+            id: redirectID,
+          },
+        },
+        auth_type: "SRN", //for now, this is hardcoded
+        User: {
+          values: userID,
+          userdata_validated: doesUserExist,
+        },
+      },
+    ];
+    const params = {
+      MessageBody: JSON.stringify(messageBody),
+      QueueUrl: queueURL,
```

Comment:
Why is the case of these properties proper case?
Why not `dateTime` instead of `DateTime`? Any particular reason?

Reply:
No reason, per se.

Reply:
I think it'll be best to convert these into camel case?
As I've commented on the lambda PR as well, would be best to use `camelCase` here. Also because you'll be accessing these values from the SQS payload na, so would have to change there as well.

https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase#:~:text=The%20JSON%20syntax%20does%20not,or%20snake_case%20should%20work%20fine. 
Read the second answer, pretty interesting!